spec/requests/api/populations_controller_index_spec.rb

```diff
@@ -0,0 +1,153 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Api::PopulationsController do
+  let(:access_token) { 'spoofed-token' }
+  let(:headers) { { 'CONTENT_TYPE': content_type }.merge('Authorization' => "Bearer #{access_token}") }
+  let(:response_json) { JSON.parse(response.body) }
+  let(:content_type) { ApiController::CONTENT_TYPE }
+
+  describe 'GET /locations_free_spaces' do
+    subject(:get_locations_free_spaces) { get '/api/locations_free_spaces', params: params.merge(date_params), headers: headers }
+
+    let(:schema) { load_yaml_schema('get_locations_responses.yaml') }
+    let(:params) { {} }
+    let(:date_from) { Date.today }
+    let(:date_to) { Date.tomorrow }
+    let(:date_params) { { date_from: date_from.to_s, date_to: date_to.to_s } }
+
+    context 'when successful' do
+      before { get_locations_free_spaces }
+
+      it_behaves_like 'an endpoint that responds with success 200'
+    end
+
+    describe 'meta data' do
+      let!(:location) { create(:location, :prison) }
+      let!(:population) { create(:population, location: location, date: date_from) }
+      let(:expected_json) do
+        {
+          data: [
+            {
+              "id": location.id,
+              "type": 'locations',
+              "attributes": {
+                "title": location.title,
+              },
+              "meta": {
```

Comment:
Shouldn't the meta be at the root? https://jsonapi.org/format/#document-meta

Reply:
It's supported both at document and resource level. Here's the relevant section - plus the gem wouldn't support it if it wasn't a thing I guess...

```
In addition, a resource object MAY contain any of these top-level members:

* attributes: an attributes object representing some of the resource’s data.
* relationships: a relationships object describing relationships between the resource and other JSON:API resources.
* links: a links object containing links related to the resource.
* meta: a meta object containing non-standard meta-information about a resource that can not be represented as an attribute or relationship.
```